ci/OWNERS

```diff
@@ -391,3 +391,13 @@ pkgs/by-name/lx/lxc*                    @adamcstephens
 /pkgs/by-name/in/installShellFiles/*     @Ericson2314
 /pkgs/test/install-shell-files/*         @Ericson2314
 /doc/hooks/installShellFiles.section.md  @Ericson2314
+
+# Xen Project Hypervisor
+/nixos/modules/virtualisation/xen*      @NixOS/xen-project
+/pkgs/by-name/xe/xen/*                  @NixOS/xen-project
+/pkgs/by-name/qe/qemu_xen/*             @NixOS/xen-project
+/pkgs/by-name/xe/xen-guest-agent/*      @NixOS/xen-project
+/pkgs/by-name/xt/xtf/*                  @NixOS/xen-project
+/pkgs/build-support/xen/*               @NixOS/xen-project
+/pkgs/development/ocaml-modules/xen*/*  @NixOS/xen-project
+/pkgs/development/ocaml-modules/vchan/* @NixOS/xen-project
```

Comment:
These would be redundant with the `maintainers` entries (maybe not the `build-support` one? not sure how pings work for those). See the comment at the top of the file.

Reply:
My reasoning was that OfBorg can't request teams for review; only individual people in said teams.

Removed.

Reply:
So, it turns out OfBorg does not request review for module changes. [This PR](https://github.com/NixOS/nixpkgs/pull/353371) was going to alter all of our systemd units, and we weren't notified. I now think it's a good idea to add the team to OWNERS, as I do not want the Xen module to be inadvertently broken because someone "wants to play minecraft with the boys".

Reply:
I am not sure that is true about ofborg, but what you would have run into with that PR anyway is the 15‐reviewer limit. Being a code owner wouldn’t have helped unless we get into zero‐sum competition for GitHub usernames that sort alphabetically highest.

(But besides a no‐op change definitionally wouldn’t break anything anyway, right?)